osu.Game/Overlays/Chat/ChannelControl/ControlItem.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.Containers;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItem : OsuClickableContainer
+    {
+        public event Action<Channel>? OnRequestSelect;
+        public event Action<Channel>? OnRequestLeave;
+
+        public int MentionCount
+        {
+            get => mention?.MentionCount ?? 0;
+            set
+            {
+                if (mention != null)
+                    mention.MentionCount = value;
+            }
+        }
```

Comment:
These sorts of properties come out way cleaner using bindable flow. Less worrying about whether the underlying thing being delegated to exists etc.

Reply:
Good point. For some reason using bindable flow for this slipped my mind even though I was using it for the selected channel state. Will change.